Source/SourceKittenFramework/WindowsError.swift

```diff
@@ -0,0 +1,32 @@
+#if os(Windows)
+import WinSDK
+
+@_transparent
```

Comment:
Why `@_transparent` instead of `@inlinable`? According to this doc I don't see why this should be transparent: https://github.com/apple/swift/blob/main/docs/TransparentAttr.md

Reply:
2 reasons:
1) less divergence from the original source of this code (https://github.com/compnerd/swift-win32)
2) This really is a macro in C and the `@_transparent` mimics the behaviour more faithfully:

```c
#define MAKELANGID(p, s) ((s) << 10 | (p))
```

I can change it if you feel strongly.

Reply:
I don’t feel strongly, just trying to understand the motivation. Thanks for sharing.